lib/links.js

```diff
@@ -1,8 +1,8 @@
 import cheerio from 'cheerio';
 
-const FIXES_RE = /(Close[ds]?|Fix(e[ds])?|Resolve[sd]?)\s*:\s*(\S+)/mgi;
+const FIXES_RE = /^\s*(Close[ds]?|Fix(e[ds])?|Resolve[sd]?)\s*:\s*(\S+)/mgi;
 const FIX_RE = /(Close[ds]?|Fix(e[ds])?|Resolve[sd]?)\s*:\s*(\S+)/i;
-const REFS_RE = /Refs?\s*:\s*(\S+)/mgi;
+const REFS_RE = /^\s*Refs?\s*:\s*(\S+)/mgi;
```

Comment:
Maybe for a separate PR, but I think we probably don't need to tolerate spaces before the colon:

```suggestion
const REFS_RE = /^\s*Refs?:\s*(\S+)/mgi;
```